### DISCUSSION TITLE
iOS 16

### DISCUSSION THREAD

**Discussion by noelhibbard** [mikebrady/shairport-sync#1480] [General]
I know I know, it's beta but I just wanted to point out that SPS-AP2 doesn't show up in iOS16. I haven't tried the AP1 build yet.

**Comment by noelhibbard** [comment]
The AP1 build isn't working on iOS 16 either. Why does Apple have to do this crap? Why can't they just open source AirPlay? There are no reasonably priced official AirPlay 2 solutions and no way am I going to invest in the little toy HomePod speakers to get music throughout my house. This is so frustrating.

**Reply by charlesomer** [reply]
I've just tested my Pi3 with the SPS docker image and it seems to work fine :/ iOS 16

**Reply by noelhibbard** [reply]
Okay that is interesting. I guess I need to do some debugging. Thanks for confirming it's just me.

**Reply by noelhibbard** [reply]
It's working now. I simply restarted my containers and now all is well. I guess I should have tried that before wasting everyone's time. Especially while Mike is on vacation.

**Reply by noelhibbard** [reply]
All my instances dropped off again but this time rather than restarting my containers I simply cycled WiFi off/on on my phone and they all came back but only for 30secs or so and they all went away again. I'm guessing this is some sort of mdns issue. I have some Rokus that support AirPlay that are also randomly dropping from the list. I'm guessing this is more of a bug in iOS 16 than anything else.
